eeg_cofitage/participants.json

```diff
@@ -0,0 +1,16 @@
+{
+  "age":{
+    "LongName":"Age",
+    "Description":"Age of a subject",
+    "Units":"year"
+  },
+  "sex":{
+    "LongName":"Sex",
+    "Description":"Sex of a subject",
+    "Levels":{
+      "n/a":"Not available",
```

Comment:
`n/a` is defined at a BIDS level, there is no need to additionally document its meaning in `.json` files

Reply:
Will do

Reply:
Done